src/entities/flags-config.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Flags used to enable or disable certain features in the sdk.
+ * @public
+ */
+export interface FlagsConfig {
+  /**
+   * If set to true, the SDK will automatically collect UTM parameters and store them as at the time of purchase.
+   * @defaultValue true
+   */
+  collectAutomaticallyUTMParamsAsMetadata?: boolean;
```

Comment:
Nitpick but maybe rename to `collectUTMParamsAsMetadataAutomatically`? Sounds a bit better to me 😅. Not a blocker though.

Reply:
Or `autocollectUtmParamsAsMetadata`?

Reply:
or simply `autoCollectUTMAsMetadata`

Reply:
Renamed to `autoCollectUTMAsMetadata`